apps/api-wallet/src/app/transactions/transactions.module.ts

```diff
@@ -1,9 +1,14 @@
 import { Module } from '@nestjs/common';
 import { TransactionsController } from './transactions.controller';
 import { TransactionsService } from './transactions.service';
+import { ConfigModule } from '@nestjs/config';
+import sampleConnectionProvider from '../database/providers/db_provider';
+import DB_Config from '../../db_config/db.config';
 
 @Module({
+  imports: [ConfigModule.forFeature(DB_Config)],
+  exports: [sampleConnectionProvider],
   controllers: [TransactionsController],
-  providers: [TransactionsService],
+  providers: [TransactionsService, sampleConnectionProvider],
```

Comment:
change this name lang to be more informative

Reply:
done